public/Register-DbatoolsConfig.ps1

```diff
@@ -56,6 +56,13 @@ function Register-DbatoolsConfig {
         PS C:\> Register-DbatoolsConfig -Module Message -Scope SystemMandatory
 
         Retrieves all configuration items of the module Message, then registers them in registry to enforce them for all users on the current system.
+
+    .EXAMPLE
+        PS C:\> Set-DbatoolsConfig -FullName sql.connection.trustcert -Value $true -PassThru | Register-DbatoolsConfig
+
+        Set the `sql.connection.trustcert` configuration to be $true, and then use the -PassThru parameter
```

Comment:
The back ticks don't render code statement on our docs page, that is only for markdown support 😉

Reply:
Gotcha - shall I remove? Or ok just to show separation?

Reply:
I'd remove and put in double-quotes or brackets if you need to call focus to it.

Reply:
Consider it done :)